frontend/ContextMenu.js

```diff
@@ -58,16 +63,39 @@ class ContextMenu extends React.Component {
     this.props.hideContextMenu();
   }
 
+  getHeight = element => {
```

Comment:
I guess it's okay. We can just combine that PR and this PR if it's easier for you.